master/getting-started/kubernetes/installation/app-layer-policy.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Installing Calico with Application Layer Policy
+canonical_url: 'https://docs.projectcalico.org/master/getting-started/kubernetes/installation/app-layer-policy'
+---
+
+Application Layer Policy for {{site.prodname}} allows you to write policies that enforce against both network layer attributes, like IP addresses or ports and application layer attributes like HTTP Method or Path. It can also enforce policy based on cryptographically secure identities, allowing you to reduce trust in your network.  See [Application Layer Policy Introduction](link me) for more information.
+
+# Kubernetes Cluster Requirements
+
+Application Layer Policy is only supported on Kubernetes, and the cluster must meet the following requirements:
+
+  * Kubernetes v1.9 or later
+  * [Mutating Webhook](https://kubernetes.io/docs/admin/admission-controllers/#mutatingadmissionwebhook-beta-in-19) support is enabled
+
```

Comment:
Great to have this information. Is there any way you could fold it into our requirements includes instead so that it shows up on the REquirements page for Kubernetes? That is what we are trying to standardize on. You can just mention that for ALP, 1.9 or later is required. Also in the Kubernetes section, you can add a new subsection called Mutating webhook and mention that to use ALP, you need to enable it. Lastly, I don't think we need to mention that ALP is only supported on K8s as this content will be inside of the K8s directory tree